File: python/jzoffer-7.py
# -*- coding:utf-8 -*-
class Solution:
    def Fibonacci(self, n):
        # write code here
        if n == 0:
            return 0
        x,a,b = 1,0,1
        while x < n:
            a,b = b,a+b
            x+=1
        return b

File: java/jzoffer-7.java
public class Solution {
    public int Fibonacci(int n) {
        if(n == 0){
            return 0;
        }
        int x = 1,a = 0,b = 1;
        int i = 0;
        int j = 0;
        while(x < n){
            i = b;
            j = a + b;
            a = i;
            b = j;
            x += 1;
        }
        return b;
    }
}